articles/WideStrings.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: default
+title: WideStrings
+abstract:
+  Some ROS2 users would like to use multi-byte characters for topic data, suitable for supporting multi-byte languages.  This article lays out the problem, and explores potential solutions to this problem.
+author: '[Chris Lalancette](https://github.com/clalancette)'
+published: true
+---
+
+- This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Background
+
+ROS2 currently supports the concept of strings, which are sequences of single-byte
+characters.  When working with ROS 2.0 topic data, some users would like to use multi-byte
```

Comment:
Maybe worth to mention that ROS2 has no encoding convention besides ASCII (7 bits).
Some of the issues you raise (agreeing on encoding) already exist when trying to use non-ASCII (but single-byte) character set.